serialization/src/main/scala/sbt/serialization/CustomPicklerUnpickler.scala

```diff
@@ -1,18 +1,23 @@
 package sbt.serialization
 
-import sbt.serialization.pickler.{
+// TODO - Why are type aliases not workign?
```

Comment:
With `sbt.protocol.CoreProtocol`, the compilation ordering was

1. scala.pickling.*
2. sbt.serialization.*
3. sbt.protocol.CoreProtocol

Now that `sbt.serialization.(package)` is a package object as well as a cake, the compilation order is: 

1. scala.pickling.*
3. sbt.serialization.CustomPicklers
3. sbt.serialization.(package)

I think.